.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - node
+cache:
+  yarn: true
+  directories:
+    - website/node_modules
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+script:
+  - cd website
+  - yarn
+  - yarn lint
+  - yarn watch -- --no-watch
```

Comment:
Yeah, the production build is considerably slower, so I used this instead to make PR feedback faster (assuming Travis is enabled for this repo). I can change it, though, if you'd still prefer `yarn build`. Just let me know.